test/sequential/xyk-pallet.api.maintenance-config.test.ts

```diff
@@ -170,7 +170,7 @@ test("check UpgradabilityOn can only be set after MaintenanceModeOn is set and M
   await checkMaintenanceStatus(false, false);
 });
 
-test("maintenance- check we can sell MGX tokens and compoundRewards THEN switch maintenanceMode to on, repeat the operation and receive error", async () => {
+test.skip("maintenance- check we can sell MGX tokens and compoundRewards THEN switch maintenanceMode to on, repeat the operation and receive error", async () => {
```

Comment:
Could you please check if there is any Sell test for maintenance? If not, simply remove the compoundRewards and level the selling. Im just wondering if we are missing some coverage by skipping the whole test and loosing that "sell MGX" on maintenance coverage.

Reply:
I didn't find, so I changed this test.